content/docs/deploy/core/upgrading.mdx

```diff
@@ -13,6 +13,52 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## 0.24.0
+
+### Breaking
+
+#### Set Authorization Header
+
+The deprecated `set_authorization_header` configuration setting has been removed. You can use the [Set Request Headers](/docs/reference/routes/headers#set-request-headers) setting to pass IdP tokens to upstream services in any header.
+
+#### Base64-encoded Certificates
+
+Previously, the `certificates` key supported base64-encoded certificates as a value. We've removed support for base64-encoded certificates for this setting and now require that you only use the relative file location.
+
+See [Certificates](/docs/reference/certificates#certificates) for more information.
+
+#### Remove Redis Storage Backend
+
+Support for Redis as a storage backend for Pomerium has been completely removed. We recommend a dedicated Postgres instance for Pomerium to provide the strongest security and performance guarantees.
```

Comment:
Should we mention that the Redis backend has been discouraged since v0.18?

```suggestion
PostgreSQL has been the recommended databroker storage backend since v0.18. Support for Redis has now been removed.
```